docs/release-notes/cloud/cloud-2023-12-DD-gh-admin-permission-not-requested.md

```diff
@@ -0,0 +1,60 @@
+---
+rss_title: Codacy release notes RSS feed
+rss_href: /feed_rss_created.xml
+---
+
+# Codacy no longer requests repository permissions for Administration on GitHub December DD, 2023
+
+!!! info "This release note applies only to GitHub"
+
+For increased security, Codacy is deprecating the usage of **repository SSH keys** for Git operations on GitHub in favour of **installation access tokens**.
+
+This change translates into an important security improvement by reducing the liability of Codacy GitHub App, as it no longer requests [read and write repository permissions for Administration](https://docs.github.com/en/enterprise-cloud@latest/rest/overview/permissions-required-for-github-apps?apiVersion=2022-11-28#repository-permissions-for-administration).
+
+## What do you need to do if you have a GitHub organization? {: id="to-do"}
+
+Make sure an organization owner [approves Codacy GitHub App updated permissions](https://docs.github.com/en/apps/using-github-apps/approving-updated-permissions-for-a-github-app) on your GitHub organization, if not done yet.
```

Comment:
Inverting the sentence here reads better to me 🤔 

```suggestion
Make sure an organization owner [approves the updated Codacy GitHub App permissions](https://docs.github.com/en/apps/using-github-apps/approving-updated-permissions-for-a-github-app) on your GitHub organization, if not done yet.
```

Reply:
My intent is to match GitHub terminology, [Approving updated permissions for a GitHub App](https://docs.github.com/en/apps/using-github-apps/approving-updated-permissions-for-a-github-app).

WDYT?

Reply:
How about an alternative that keeps the meaning and still sticks to the GH terminology?

```suggestion
Make sure an organization owner [approves the updated permissions for the Codacy GitHub App](https://docs.github.com/en/apps/using-github-apps/approving-updated-permissions-for-a-github-app) on your GitHub organization, if not done yet.
```